### PR TITLE
[TASK] Fix some code style issues

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

**Comment by kdambekalns** [comment]
With this (and an upmerge), both the 2.0 and the master branch turn green on StyleCI, we could then enable the checks on those as well.

File: TYPO3.Neos/Classes/TYPO3/Neos/Routing/Exception/InvalidDimensionPresetCombinationException.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 namespace TYPO3\Neos\Routing\Exception;
 
- /*    /*
+/*    /*
```
Comment (kitsunet) [inline TYPO3.Neos/Classes/TYPO3/Neos/Routing/Exception/InvalidDimensionPresetCombinationException.php]: The second opening comment can be removed...

**Comment by kitsunet** [comment]
:+1:  Waiting for travis then merging

**Comment by kdambekalns** [comment]
Thanks for spotting this, I was too focused on actual whitespace changes, it seems…

**Pull request merged**